backend/src/modules/cards/services/create.card.service.ts

```diff
@@ -18,8 +18,15 @@ export default class CreateCardServiceImpl implements CreateCardService {
     card: CardDto,
     colIdToAdd: string,
   ) {
-    card.createdBy = userId;
-    card.items[0].createdBy = userId;
+    card.createdBy = userId.toString();
+
+    if (card.items.length === 0) {
```

Comment:
You can do this, or you can create an util to verify if a value is empty, you can reach me to help you with it 😉 

```suggestion
    if (!card.items.length) {
```